samples/XCT.Sample/Pages/Views/SegmentedViewPage.xaml

```diff
@@ -0,0 +1,193 @@
+                            MaximumTrackColor="DarkBlue"
+                            ThumbColor="{StaticResource NormalButtonBackgroundColor}"
+                            ValueChanged="TopRightSlider_ValueChanged"/>
+
+                    <StackLayout Grid.Row="2" Grid.Column="1">
+                        <Label VerticalOptions="Center"
+                           HorizontalTextAlignment="Center"
+                           Text="{Binding Source={x:Reference Slider},
+                            Path=Value,
+                            StringFormat='Corner Radius: {0}'}" />
+                        <Slider
+                            x:Name="Slider"
+                            Minimum="0"
+                            Maximum="20"
+                            HeightRequest="40"
+                            MinimumTrackColor="Blue"
+                            MaximumTrackColor="DarkBlue"
+                            ThumbColor="{StaticResource NormalButtonBackgroundColor}"
+                            ValueChanged="Slider_ValueChanged"/>
+                    </StackLayout>
+
+                    <Slider Grid.Row="3" Grid.Column="0"
+                            x:Name="BottomLeftSlider"
+                            Minimum="0"
+                            Maximum="20"
+                            HeightRequest="40"
+                            MinimumTrackColor="Blue"
+                            MaximumTrackColor="DarkBlue"
+                            ThumbColor="{StaticResource NormalButtonBackgroundColor}"
+                            ValueChanged="BottomLeftSlider_ValueChanged"/>
+
+                    <Slider Grid.Row="3" Grid.Column="2"
+                            x:Name="BottomRightSlider"
+                            Minimum="0"
+                            Maximum="20"
+                            HeightRequest="40"
+                            MinimumTrackColor="Blue"
+                            MaximumTrackColor="DarkBlue"
+                            ThumbColor="{StaticResource NormalButtonBackgroundColor}"
+                            ValueChanged="BottomRightSlider_ValueChanged"/>
+                </Grid>
+            </Frame>
+
+            <StackLayout Grid.Row="2" Spacing="10">
+                <toolkit:SegmentedView
+                    x:Name="TextSegments"
+                    DisplayMode="{Binding DisplayMode}"
+                    ItemsSource="{Binding Options}"
+                    SelectedItem="{Binding SelectedItem}"
+                    SelectedIndex="0"/>
+
+                <StackLayout Orientation="Horizontal">
+
+                <StackLayout
+                    HorizontalOptions="FillAndExpand"
+                    Spacing="8"
+                    Padding="20">
+                    <Button Text="Add"
+                            Clicked="Add_Clicked"
+                            Command="{Binding AddTextCommand}"
+                            CommandParameter="0"/>
```

Comment:
This looks redundant, was it intended ?

Reply:
Yea. For some reason, couldn't get the command to work. So did the Clicked. Cleaning this up.